drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -219,30 +219,30 @@ matterGeneric:
   - id: "matter/air-quality-sensor"
     deviceLabel: Matter Air Quality Sensor
     deviceTypes:
-      - id: 0x002C
+      - id: 0x002C # Air Quality Sensor
```

Comment:
Thanks for adding these comments! I'll make sure I include them on future generic fingerprints too :+1: